Maths/PrimeNumber.js

```diff
@@ -0,0 +1,36 @@
+/*
+    author: Theepag
+    license: GPL-3.0 or later
+
+    This script will check weather given number is a prime number or not.
+
+    More about prime number:
+        https://en.wikipedia.org/wiki/Prime_number
+*/
+
+function test_prime(n) {
+    // Check `n` is a prime number or not.
+    'use strict'
+    if (n===1) {
+        return false;
+    }
+    else if(n === 2) {
+        return true;
+    }
+    else {
+        for(var x = 2; x < n; x++) {
```

Comment:
You could speed up the code by going only up to sqrt(n)
`for(let x = 2; x < n**0.5;  x++)`